### DISCUSSION TITLE
Source for "Jellyfin TV Show by Date"

### DISCUSSION THREAD

**Discussion by buzquin** [jmbannon/ytdl-sub#823] [Q&A]
I'm running into what appear to be dependency issues with this pre-built preset, so I want to look inside. The source is not with the code for the other presets. Where can I find it? Thanks.

**Comment by jmbannon** [comment]
Hey @buzquin , you can find all the relevant fields here: https://github.com/jmbannon/ytdl-sub/blob/master/src/ytdl_sub/prebuilt_presets/tv_show/episode.yaml#L5
